data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -722,6 +722,30 @@
     ],
     "flags": [ "ALLOW_ROTTEN" ]
   },
+  {
+    "type": "recipe",
+    "result": "soap_flakes",
+    "category": "CC_CHEM",
+    "subcategory": "CSC_CHEM_CHEMICALS",
+    "skill_used": "cooking",
+    "difficulty": 1,
```

Comment:
I think cutting up a bar of soap is trivial enough to have a difficulty of 0.

Reply:
ok i missed that, my bad.